test/integration/protocol_integration_test.cc

```diff
@@ -3265,9 +3268,12 @@ TEST_P(ProtocolIntegrationTest, HeaderOnlyBytesCountUpstream) {
   useAccessLog("%UPSTREAM_WIRE_BYTES_SENT% %UPSTREAM_WIRE_BYTES_RECEIVED% "
                "%UPSTREAM_HEADER_BYTES_SENT% %UPSTREAM_HEADER_BYTES_RECEIVED%");
   testRouterRequestAndResponseWithBody(0, 0, false);
-  expectUpstreamBytesSentAndReceived(BytesCountExpectation(167, 38, 136, 18),
-                                     BytesCountExpectation(120, 13, 120, 13),
-                                     BytesCountExpectation(116, 5, 116, 3));
+  const size_t wire_bytes_received =
+      (GetParam().http2_implementation == Http2Impl::Oghttp2) ? 10 : 13;
```

Comment:
For my understanding, why does this change based on impl?

Reply:
The oghttp2 implementation will compress some header fields that nghttp2 does not, such as ":path" and "content-length".